src/reporter.ts

```diff
@@ -366,8 +366,8 @@ export class RPReporter implements Reporter {
 
     // create step
     if (parentSuiteObj) {
-      const { isCodeRefEnabled } = this.config;
-      const codeRef = getCodeRef(test, test.title, playwrightProjectName, isCodeRefEnabled);
+      const { includePlaywrightProjectNameToCodeReference } = this.config;
+      const codeRef = getCodeRef(test, test.title, includePlaywrightProjectNameToCodeReference && playwrightProjectName);
```

Comment:
The function parameter name `pathToExclude`, so it is should receive `playwrightProjectName` by default or nothing in case of ` includePlaywrightProjectNameToCodeReference` is true.